### PR TITLE
changing name

### LINKED ISSUE #1: Building hangman game without drawing
_I am try ing to build a **hangman game** but without the hangman, if I finish this task I will try to incorporate the drawing into the script._

### PR DESCRIPTION
**Pull request by michalinadengusiak**
_I have completed task_ Fixes #1  _!!!!!_

@FraserGreenroyd 

image
_How can we/ Do we need to solve the_ **red lines**?

### CODE REVIEW

File: Python scripts/Python Hangman game withoutdrawing.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    guess = input('Please give in a character,')
+    guesses += guess
+    if guess not in word:
+        turns -= 1
+        print('Wrong!')
+        print('You have', turns, 'turns')
+        if turns == 0:
+            print('YOU LOSE!!!!!!')
+
+
```
Comment (FraserGreenroyd) [inline Python scripts/Python Hangman game withoutdrawing.py:61]: There are no comments in this file for maintainers to later use to remind anyone of what the desired functionality was here.

I'd like to see some comments and some slightly better formatting - there's a lot of blank lines included at the top of the file and not very many in the core `while` loop. I'd recommend no more than 1 blank line between code, but to add some after the empty `print` statement.

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
Functionality looks ok, just some formatting and maintaining comments needed to do it right from the off :smile:

**Comment by FraserGreenroyd** [comment]
Also, if you wish to close #1 when this PR is resolved, you need to use the `Fixes` keyword - e.g. `Fixes #1`

Additionally, even though I have provided a review requesting changes it is still allowing us to merge the PR without the review being done. You need to add branch protection to the repository. This can be found by going to the settings -> branch protection and setting some rules for the `master` branch. :smile:

**Comment by michalinadengusiak** [comment]
@FraserGreenroyd 
Finished your second comment, one to go!

**Comment by michalinadengusiak** [comment]
@FraserGreenroyd 
Fixed all your comments, please review again :stuck_out_tongue_winking_eye:

**Comment by FraserGreenroyd** [comment]
@michalinadengusiak I'm not seeing any changes to the actual `.py` file that I commented on? Did you push the changes for that file or just the notebook files?

**Review by FraserGreenroyd — APPROVED** [review]
Great work, comments resolved, happy to be merged :smile:

**Comment by michalinadengusiak** [comment]
@FraserGreenroyd 
Which of the three options should I use?

**Comment by FraserGreenroyd** [comment]
@michalinadengusiak all three will work, it depends entirely on what you want to achieve.

I would recommend strongly that you **never** squash and merge because it removes some commit history.

Either merge it, or rebase and merge it, both are equally good though merge provides additional commit history.

Rebase and merge is what your dad and I have to do at work, but the final choice is up to you :smile:

**Pull request merged**